.github/workflows/api.yaml

```
@@ -5,22 +5,46 @@ on:
+            ${{ runner.os }}-uv-${{ hashFiles('**/pyproject.toml') }}-
+            ${{ runner.os }}-uv-
+
       - name: Install dependencies
-        run: devbox run -- uv sync
+        if: steps.cache-dependencies.outputs.cache-hit != 'true'
+        run: uv sync
+
       - name: Check dependencies
-        run: devbox run -- uv run ./scripts/check_dependencies.py
+        run: uv run --active ./scripts/check_dependencies.py
+        with:
```

actionlint (1.7.7)

this step is for running shell command since it contains at least one of "run", "shell" keys, but also contains "with" key which is used for running action

(syntax-check)